app/controllers/stories/activities_controller.rb

```diff
@@ -4,6 +4,6 @@ def index
     @story = @project.stories.find(params[:story_id])
     @activities = policy_scope(Activity).by_story(@story.id)
 
-    render json: @activities
+    render json: @activities.as_json(include: :user)
```

Comment:
Is this needed? V1 board shows the author without this change, maybe we can simplify it somehow so we don't need to send in the payload some data that we already have in  the frontend.